### PR TITLE
Font Updates

### PR DESCRIPTION
**Pull request by rjspiker**
Preview: https://consul-fonts.netlify.com

This PR accomplishes the following:
- Add newly approved fonts (Gilmer, Metro Sans, Deja Vu) and remove deprecated fonts
- Global typography updates (font-size, line-height, etc), adding `.g-type` classes shared across sites
- Color updates to match color scheme with other Hashicorp sites
- Button style overhaul

### CODE REVIEW

File: website/source/assets/stylesheets/_buttons.scss [hunk 1]
```diff
@@ -1,37 +1,118 @@
 .button {
-  background: $button-background;
-  border: 1px solid $button-font-color;
-  box-shadow: 3px 4px 0 rgba(0,0,0,0.1);
-  color: $button-font-color;
+  background: $consul;
+  // border: 1px solid rgba(#1d1e23, 0.2);
```
Comment (roaks3) [inline website/source/assets/stylesheets/_buttons.scss]: Nit: seems like just a leftover line?

File: website/source/assets/stylesheets/_fonts.scss [hunk 2]
```diff
@@ -0,0 +1,83 @@
+/* Web Fonts */
+@font-face {
+  font-family: "klavika-web";
```
Comment (roaks3) [inline website/source/assets/stylesheets/_fonts.scss]: Might have missed it, but do we still need `klavika`?

**Review by roaks3 — APPROVED** [review]
Code LGTM, just some minor comments

I also tried to click around to the different pages in the preview to make sure everything still looked okay, and did not notice any visual issues.

**Review by zchsh — APPROVED** [review]
This is amazing! 💯

It seems like Klavika & Open Sans are still being loaded from Adobe Typekit - Klavika looks like it's needed for the "fits into the HashiCorp suite" thing in the top 'mega nav' bar, not sure where Open Sans is needed. That being said I'm not super familiar with the project setup here, I guess all that mega-nav .erb is in https://github.com/hashicorp/middleman-hashicorp, but the typekit stuff is being loaded from the layout here? Also I don't think we have a design alternative for the "HashiCorp Suite" piece, which does seem like it should be in Klavika. So marking as approved because that's the only thing I could find off when browsing through the preview - and based on all the other great work done here, it's definitely feels like a minor thing we can defer.

**Comment by rjspiker** [comment]
@zchsh you're completely right.  It looks like it still does get loaded in typekit, and that it's still needed in the `mega_nav`.  I _think_ that mega_nav is handled in the `middleman-hashicorp` gem.  Because of this I also agree that this is probably something we tackle later.

**Comment by rjspiker** [comment]
Thank you for the reviews!  Going to open a PR upstream.

**Pull request closed without merging**